packages/appium/lib/cli/parser.js

```diff
@@ -280,6 +293,50 @@ class ArgParser {
       }
     }
   }
+
+  /**
+   * Add subcommand and sub-sub commands for 'setup' subcommand.
+   * @param {import('argparse').SubParser} subParser
+   */
+  static _addSetupToParser(subParser) {
+    const setupParser = subParser.add_parser('setup', {
+      add_help: true,
+      help: `Install latest official drivers and/or plugins for specific usage ` +
+        `if APPIUM_HOME has no drivers and plugins. Default is 'mobile' set.`,
+    });
+
+    ArgParser._patchExit(setupParser);
+    const extSubParsers = setupParser.add_subparsers({
+      dest: `setupCommand`,
+    });
+
+    const parserSpecs = [
+      {
+        command: SUBCOMMAND_MOBILE,
+        help: `Install latest drivers for Android and iOS '${_.join(MOBILE_DRIVERS, ',')}' ` +
+          `if APPIUM_HOME has no drivers.`
+      },
+      {
+        command: SUBCOMMAND_BROWSER,
+        help: `Install latest drivers for mobile and desktop browsers '${_.join(BROWSER_DRIVERS, ',')}' ` +
+          `if APPIUM_HOME has no drivers.`
+      },
+      {
+        command: SUBCOMMAND_DESKTOP,
+        help: `Install latest drivers for desktop app '${_.join(DESKTOP_APP_DRIVERS, ',')}' ` +
+          `if APPIUM_HOME has no drivers.`
+      },
+      {
+        command: SUBCOMMAND_RESET,
+        help: `Delete contents in APPIUM_HOME.`
```

Comment:
Perform full reset by deleting all installed drivers and plugins. The server config will be reset to its initial state.